docs/Resources/resources.md

```diff
@@ -26,13 +21,23 @@ More information on running Geppetto is [available online](http://geppetto.org).
 
 [The project page for Geppetto](/Projects/geppetto/) has information about getting involved in its development with OpenWorm.
 
+### Sibernetic
+
+Sibernetic is the code base that currently implements the crawling model. Sibernetic is a C++ / Python code base by Palyanov, Khayrulin and Vella that has been created expressly for the purpose of doing research and building the model quickly.
+
+![image](http://i.imgur.com/KSWjCaW.jpg)
+
+More information on running Sibernetic is [available online](http://sibernetic.org).
+
+[The project page for Sibernetic](/Projects/sibernetic/) has information about getting involved with its development.
+
```

Comment:
is Sibernetic meant to be before or after Geppetto?

Reply:
I wouldn't think it matters.